integration-tests/build.gradle

```diff
@@ -24,7 +24,11 @@ configure(subprojects.findAll { it.name.startsWith("alfresco-") }) {
 
     apply plugin: 'java'
 
-    def alfrescoVersion = project.name - 'alfresco-'
+    if (project.name.contains("enterprise") && !project.hasProperty("enterprise")) {
+        return
+    }
+
+    def alfrescoVersion = project.name[-2..-1]
```

Comment:
I wasn't familiar with this syntax. 

Isn't this a function we need in more places ? How is that accomplished there ?

Reply:
Groovy allows to substring with negative indices.

It requires a naming convention for the integration-test subprojects where the last two characters represent the Alfresco version. E.g. `alfresco-enterprise-60`